packages/menu/src/MenuItem.tsx

```diff
@@ -122,54 +118,34 @@ interface SharedMenuItemProps {
    */
   size?: Size;
 
-  ref?: React.Ref<any>;
-}
-
-interface LinkMenuItemProps extends HTMLElementProps<'a'>, SharedMenuItemProps {
-  href: string;
-}
-
-interface ButtonMenuItemProps
-  extends HTMLElementProps<'button'>,
-    SharedMenuItemProps {
-  href?: null;
-}
-
-type CustomMenuItemProps = SharedMenuItemProps & {
-  as: React.ElementType<any>;
-  [key: string]: any;
-};
-
-type MenuItemProps =
-  | LinkMenuItemProps
-  | ButtonMenuItemProps
-  | CustomMenuItemProps;
+  /**
+   * className applied to  `li` element
+   */
+  className?: string;
 
-function usesCustomElement(props: MenuItemProps): props is CustomMenuItemProps {
-  return (props as any).as != null;
-}
+  /**
+   * Content to appear inside of `<MenuItem />` component
+   */
+  children?: React.ReactNode;
```

Comment:
Ditto here – isn't this included in the types for ExtendableBox?

Reply:
Chatted offline!